docs/content/cli/plugins_uninstall.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "porter plugins uninstall"
+slug: porter_plugins_uninstall
+url: /cli/porter_plugins_uninstall/
+---
+## porter plugins uninstall
+
+Uninstall a plugin
+
+### Synopsis
+
+Uninstall a plugin
+
+```
+porter plugins uninstall NAME [flags]
+```
+
+### Examples
+
+```
+  porter mixin uninstall azure
```

Comment:
Problem with copy/paste :)

Reply:
oops! Thank you for catching, I will review all of the documentation I copy/pasted for that error.